bootstrap-gruntwork-installer.sh

```diff
@@ -132,7 +132,9 @@ function get_os_arch {
 function get_os_arch_gox_format {
   local -r arch=$(get_os_arch)
 
-  if string_contains "$arch" "64"; then
+  if string_contains "$arch" "arm64"; then
+    echo "arm64"
+  elif string_contains "$arch" "64"; then
```

Comment:
Should we be more specific when sniffing for `x86_64` and fail explicitly on an unsupported architecture?

Reply:
We could, but this has been working for a long time that I think it's ok to keep it as is for now.

Reply:
Roger!